### PR TITLE
Update examples for consistency

### PR DESCRIPTION
**Pull request by dibyom**
# Changes

There are a few different updates to the examples folder in this PR.

1. `examples/README.md` described a single example while there were also sub-folders containing other Triggers examples. I moved the example described in `examples/README.md` to its own folder to be consistent with the other examples (embedded-triggers)

2.  I modified `examples/README.md` with a more generic Readme that now contains instructions on invoking triggers locally as well as how to look for the resources that Triggers creates (and logs in the EL when things go wrong).

3. I also fixed the ingress e2e test that was being flaky.

Part of #1067

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#tests) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#docs) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commits)
- [ ] Release notes block has been filled in or deleted (only if no user facing changes)

_See [the contribution guide](https://github.com/tektoncd/triggers/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

### CODE REVIEW

File: examples/embedded-trigger/README.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+In this example you will use Triggers to create a PipelineRun and
+PipelineResource that simply clones a GitHub repository and prints a couple of
+messages.
+
+1. Create the resources for the example
+
+```sh
+kubectl apply -f rbac.yaml
+kubectl apply -f riggertemplate.yaml
```
Comment (wlynch) [inline examples/embedded-trigger/README.md:13]:
```suggestion
kubectl apply -f triggertemplate.yaml
```

**Review by wlynch — COMMENTED** [review]
/lgtm

File: examples/embedded-trigger/README.md (hunk 1 above)
Comment (dibyom) [reply examples/embedded-trigger/README.md:13]: 🤦

**Review by wlynch — APPROVED** [review]
/lgtm

**Pull request merged**